install.sh

```diff
@@ -15,61 +13,52 @@ apt-get --yes install epoptes epoptes-client
 
 
 #Adding vagrant user to group epoptes
-
 gpasswd -a ${SUDO_USER:-$USER} epoptes
 
 #Updating kernel
-
 echo 'IPAPPEND=3' >> /etc/ltsp/update-kernels.conf
 /usr/share/ltsp/update-kernels
 
 #configure dnsmasq
-
 ltsp-config dnsmasq
 
 #Creating lts.conf
-
 ltsp-config lts.conf
 
 #Installing additional software
-
 apt-get --yes install edubuntu-desktop
 apt-get --yes install ubuntu-edu-preschool ubuntu-edu-primary ubuntu-edu-secondary ubuntu-edu-tertiary
 
 #Installing ltsp-manager
-
 add-apt-repository ppa:ts.sch.gr
 apt-get --yes update
 apt-get --yes install ltsp-manager
 
 #Creating client image
-
 ltsp-update-image --cleanup /
 
-#setting correct dhcp-range
-sed -i "s/dhcp-range=192.168.67.20,192.168.67.250,8h/dhcp-range=192.168.1.20,192.168.1.250,8h/g" /etc/dnsmasq.d/ltsp-server-dnsmasq.conf
-service dnsmasq restart	
+#source setting.sh
+source /vagrant/settings.sh
+NET=$(echo $NETWORK | cut -d'.' -f 3)
+
+#setting dhcp-range
+sed -i 's/dhcp-range=192.168.67.20,192.168.67.250,8h/dhcp-range=192.168.'"$NET"'.20,192.168.'"$NET"'.250,8h/g' /etc/dnsmasq.d/ltsp-server-dnsmasq.conf
```

Comment:
This command can be more readable like this:
```
sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf \
    -e "/^dhcp-range=/ c dhcp-range=${NETWORK}.20,${NETWORK}.250,8h"
```

Writing beautiful and readable code is an art that you should try actively to learn.

Reply:
This changes multiple lines. will be using 
```
sed -i /etc/dnsmasq.d/ltsp-server-dnsmasq.conf -e "/8h$/ c dhcp-range=${NETWORK}.20,${NETWORK}.250,8h"
```
instead to uniquely identify this particular line. I will learn how to write code readable